test/checkHotNet2Consensus.sh

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/env bash
+
+# Setup directories.
+hotnet2=../
+hotnet2_test=/tmp/hotnet2_consensus_test
+data=$hotnet2_test/data
+results=$hotnet2_test/results
+
+# Setup parameters.
+beta=0.4
+num_network_permutations=100
+num_heat_permutations=200
+num_cores=8
```

Comment:
Do the number of heat/network permutations actually need to be set this high to check the code?